src/xenia/cpu/processor.h

```diff
@@ -46,10 +49,11 @@ enum class Irql : uint32_t {
 
 class Processor {
  public:
-  Processor(Memory* memory, ExportResolver* export_resolver,
+  Processor(Emulator* emulator, Memory* memory, ExportResolver* export_resolver,
```

Comment:
ah - good to drop now then - it took me awhile to remove the cycles having it introduced between the modules